src/Enterspeed.Cli/Commands/BaseCommandHandler.cs

```diff
@@ -24,4 +26,11 @@ public bool GetConfirmation()
             return true;
         return false;
     }
-}   
\ No newline at end of file
+
+    protected async Task<GetEnvironmentsResponse> GetEnvironmentToDeployTo(string environmentName, IMediator mediator)
```

Comment:
I don't really think that this method belongs here. Either duplicate it in the two methods calling it, or create a mediator request/handler that return environment based on name.

Reply:
@jthind Updated :)